### PR TITLE
Update oper.xml for mutable digraphs

### PR DESCRIPTION
**Pull request by flsmith**
Updates the documentation in oper.xml to be true

### CODE REVIEW

File: doc/oper.xml [hunk 1]
```diff
@@ -116,33 +116,40 @@ true]]></Example>
...
+    If <A>digraph</A> is a digraph without multiple edges, and <A>edges</A> is a
+    list of pairs of vertices of <A>digraph</A> (the entries of each pair
+    corresponding to the source and the range of an edge, respectively),
+    then <C>DigraphReverseEdges</C> returns a digraph constructed from
     <A>digraph</A> by reversing the orientation of every edge specified by
-    <A>edges</A>.  If only one edge is to be reversed, then
-    <C>DigraphReverseEdge</C> can be used instead.  In this case, the second
-    argument should just be a single vertex-pair or a single position. <P/>
+    <A>edges</A>. If only one edge is to be reversed, then
+    <C>DigraphReverseEdge</C> can be used instead  In this case, the second
```
Comment (james-d-mitchell) [inline doc/oper.xml]: Missing full stop.

File: doc/oper.xml [hunk 2]
```diff
@@ -116,33 +116,40 @@ true]]></Example>
...
-        (the entries of each pair corresponding to
-        the source and the range of an edge, respectively),
-      </Item>
-    </List>
-
-    then <C>DigraphReverseEdges</C> returns a new digraph constructed from
+    If <A>digraph</A> is a digraph without multiple edges, and <A>edges</A> is a
+    list of pairs of vertices of <A>digraph</A> (the entries of each pair
+    corresponding to the source and the range of an edge, respectively),
+    then <C>DigraphReverseEdges</C> returns a digraph constructed from
```
Comment (james-d-mitchell) [inline doc/oper.xml]: What about the 3-arg version?

File: doc/oper.xml [hunk 3]
```diff
@@ -177,21 +184,26 @@ gap> DigraphReverseEdge(gr, [7, 19]);
     The <Ref Oper="DigraphVertexLabels"/> of <A>digraph</A> will not be retained
     in the returned digraph. <P/>
 
+    If <A>digraph</A> belongs to <Ref Filt="IsMutableDigraph"/>, then relabeling
```
Comment (james-d-mitchell) [inline doc/oper.xml]: Spelling

File: doc/oper.xml [hunk 4]
```diff
@@ -221,17 +233,22 @@ true
     in the action of a permutation on the vertices of a digraph, then you can
     use <C>OnDigraphs</C> instead of <C>OnMultiDigraphs</C>.
 
+    If <A>digraph</A> belongs to <Ref Filt="IsMutableDigraph"/>, then relabeling
```
Comment (james-d-mitchell) [inline doc/oper.xml]: Spelling

File: doc/oper.xml [hunk 5]
```diff
@@ -410,11 +501,64 @@ gap> func := function(n)
...
+  <Oper Name="DigraphRemoveEdge" Arg="digraph, edge"/>
+  <Oper Name="DigraphRemoveEdge" Arg="digraph, src, ran"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>digraph</A> is a digraph with no multiple edges and <A>edge</A> is a
+    pair of vertices of <A>digraph</A>, or <A>src</A> and <A>ran</A> are
+    vertices of <A>digraph</A>, then this operation returns a digraph
+    constructed from <A>digraph</A> by removing the edge specified by
+    <A>edge</A> or <A>[src, ran]</A>. <P/>  
+
```
Comment (james-d-mitchell) [inline doc/oper.xml]: Double blank line

**Comment by wilfwilson** [comment]
Thanks @flsmith - I made some final tweaks before merging. There were a couple of points where the spiel about mutable/immutable was duplicated.

**Pull request merged**